src/libDirectoryService/DSBlockPreProcessing.cpp

```diff
@@ -1027,7 +1027,7 @@ bool DirectoryService::RunConsensusOnDSBlockWhenDSBackup() {
   }
 
 #ifdef VC_TEST_VC_PRECHECK_1
-  if (m_consensusMyID == 3) {
+  if (m_consensusMyID == 3 && m_mediator.m_currentEpochNum == 2) {
```

Comment:
What's the meaning of "3" and "2"?

Reply:
same as above

Reply:
Why the vc test have been scope to so specific test?

Reply:
I just wanted one node to die. I can revert it however